contracts/ArchetypeLogic.sol

```diff
@@ -162,6 +163,11 @@ library ArchetypeLogic {
     }
 
     uint256 cost = price * numTokens;
+    
+    if(curSupply >= 10000 && invite.interval == 0){ //Apply the linear curve after minted 10000 nfts
```

Comment:
this isn't a request for a change, but I'm trying to find a way to avoid hardcoding 10,000 here. 

This threshold should either be dynamic, ie passed in the invite list, or made a function of max supply.

That way we can make this contract available to other collections with different requirements. 

@kneelsdev what do you think?